aws/resource_aws_codedeploy_app.go

```diff
@@ -26,6 +27,16 @@ func resourceAwsCodeDeployApp() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"compute_platform": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					codedeploy.ComputePlatformServer,
+					codedeploy.ComputePlatformLambda,
+				}, false),
+				Default: codedeploy.ComputePlatformServer,
```

Comment:
We should check the default value in `TestAccAWSCodeDeployApp_basic` as well (when not provided in the test configuration):

```go
resource.TestCheckResourceAttr("aws_codedeploy_app.foo", "compute_platform", "Server"),
```